language/move-lang/src/cfgir/liveness/mod.rs

```diff
@@ -294,12 +294,20 @@ mod last_usage {
                     match display_var(v.value()) {
                         DisplayVar::Tmp => (),
                         DisplayVar::Orig(v_str) => {
-                            let msg = format!(
-                                "Unused assignment or binding for local '{}'. Consider removing \
-                                 or replacing it with '_'",
-                                v_str
-                            );
-                            context.error(vec![(l.loc, msg)]);
+                            if !v.starts_with_underscore() {
+                                let verb = if context.is_resourceful(&v) {
+                                    // do not suggest prefixing for resource bindings
+                                    "replacing"
```

Comment:
I think this is unnecessary, replacing it with `_` if it is resourceful will result in an error, so it might be okay to just say "replacing / prefixing" always. 

I might be a bit more explicit about prefixing so maybe:

"Unused assignment or binding for local '{:v_str}'. Consider removing, replacing with '_', or prefixing with '_' e.g. '_{:v_str}'"

That might be a bit wordy, but I think it is good to squeeze in the prefix example somehow